services/ocr/main.py

```diff
@@ -1,109 +1,37 @@
-from flask import Flask, request, jsonify
-from flask_cors import CORS
-from flasgger import Swagger
-import numpy as np
-import os, ocr, cv2
-
-app = Flask(__name__)
-app.secret_key = os.environ["SECRET_KEY"]
-CORS(app)
 
-swagger_config = {
-    "headers": [
-    ],
-    "specs": [
-        {
-            "endpoint": 'apispec_1',
-            "route": '/apispec_1.json',
-            "rule_filter": lambda rule: True,
-            "model_filter": lambda tag: True,
-        }
-    ],
-    "static_url_path": "/flasgger_static",
-    "swagger_ui": True,
-    "specs_route": "/docs"
-}
-swagger = Swagger(app, config=swagger_config)
+from ocr import Engine
+import numpy as np
+import pika, sys, os, json, base64, cv2
 
-@app.route(f"/api/status", methods=["GET"])
-def status():
-    """
-    GET to check the status of the application
-    ---
-    responses:
-      200:
-        description: OK
-        schema:
-          id: Status
-          properties:
-            status:
-              type: string
-              description: The status of the application
-              default: ok
-    """
-    return jsonify(status="OK")
+RABBITMQ_HOST = "127.0.0.1"
```

Comment:
V dockeru bude muset být `rabbitmq` místo `127.0.0.1`, zkus to kdyžtak udělat přes flag, jako že když spustí `python main.py -docker` tak se podle toho změní tahle adresa